pyproject.toml

```diff
@@ -10,10 +10,11 @@ authors = [
 
 [tool.poetry.dependencies]
 python = "^3.12"
-sqlalchemy = "^2.0.22"
+sqlalchemy = {extras = ["mypy"], version = "^2.0.23"}
 alembic = "^1.12.1"
 boto3 = "^1.28.76"
 psycopg2 = "^2.9.9"
+requests = "^2.31.0"
```

Comment:
`requests` library for handing GET Requests

added `mypy` to sqlalchemy extras, as that was the recommended way for ensure typing  support for SQLAlchemy, but since we already had `mypy` installed I don't think this actually does anything